.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  main:
+    name: Create Protobuf generated files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup GO
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Setup Typescript
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.17.1
+      - run: npm install -g typescript
```

Comment:
Do we need to pin this to a good version?

Reply:
yeah good shout. moved it to v16. its what they use for proto-as-rust and its building fine